pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -48,6 +50,14 @@ def connect(
         """Connects this plugin to the accelerator and the training process."""
         return model, optimizers, lr_schedulers
 
+    @property
```

Comment:
Why do we define these properties for this class?

Reply:
Shouldn't we remove that from the accelerator instead of adding the property here?

It's really weird to have this detail from one of the plugins leak into the common interface

Reply:
I moved this implementation back to the existing trainer properties which is currently the main way these are accessed  everywhere. Then they don't have to be defined on the base  precision class.